sdk/storage/sample/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+if (BUILD_CURL_TRANSPORT)
```

Comment:
What's this for?

Reply:
see https://github.com/Azure/azure-sdk-for-cpp/blob/711fe5f8c624d9b41cda7d6036b087fb445829e8/CMakeLists.txt#L9